loops-and-arrays.js

```diff
@@ -36,6 +44,9 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+for (let i = 0; i < squares.length; i++) {
+    console.log(Math.pow(squares[i],3));
```

Comment:
Hieronder de code zoals in de uitwerkingen 

for (let i = 0; i < squares.length; i++) {
  squares[i] = squares[i] * squares[i] * squares[i];
}

console.log(squares);